src/main/java/IP/duke/main/Storage.java

```diff
@@ -0,0 +1,116 @@
+package IP.duke.main;
+
+import IP.duke.task.Deadline;
+import IP.duke.task.Event;
+import IP.duke.task.Task;
+import IP.duke.task.Todo;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Represents a file to store data, operations to write on the file.
+ * 
+ * @author Gordon
+ * @version CS2103T, Semester 2
+ */
+public class Storage {
+    private String filePath;
+    private String fileName;
+    private File file;
+    private FileWriter fileWriter;
+    
+    /**
+     * Class constructor.
+     * 
+     * @param filePath the path from the project directory to the storage file.
+     * @param fileName the name of the file.
+     */
+    public Storage(String filePath, String fileName) {
+        this.filePath = filePath;
+        this.fileName = fileName;
+    }
+
+    /**
+     * Loads all the tasks saved in the storage folder into a tasks arraylist.
+     * 
+     * @return an arraylist of tasks.
+     * @throws IOException
+     */
+    public  ArrayList<Task> load() throws DukeException {
+        ArrayList<Task> tasks = new ArrayList<>();
+        try {
+            File fileDirectory = new File(filePath);
+            if(!fileDirectory.exists()) {
```

Comment:
Should there be a space between `if` and `(`? I also spotted this in different places.
```suggestion
            if (!fileDirectory.exists()) {
```